.github/workflows/build-windows-selfhosted.yml

```diff
@@ -77,4 +77,4 @@ jobs:
       - name: Smoke test
         working-directory: fstar
         run: |
-          .\bin\fstar.exe --already_cached "*,-Hello" ${{ github.workspace }}\fstartest\examples\hello\Hello\Hello.fst -f -d
+          .\bin\fstar.exe --already_cached *,-Hello ${{ github.workspace }}\fstartest\examples\hello\Hello\Hello.fst -f --debug_all --debug_all_modules
```

Comment:
Note that `--debug_all` is really excessive, but I don't mind keeping it if you suspect it will help debug possible failures here.